dali/pipeline/data/types.h

```diff
@@ -585,6 +584,27 @@ DALI_REGISTER_TYPE(std::vector<float>, DALI_FLOAT_VEC);
 DALI_REGISTER_TYPE(std::vector<TensorLayout>, DALI_TENSOR_LAYOUT_VEC);
 DALI_REGISTER_TYPE(std::vector<DALIDataType>, DALI_DATA_TYPE_VEC);
 
+
+inline std::string to_string(DALIDataType dtype) {
+  if (const char *builtin = GetBuiltinTypeName(dtype))
+    return builtin;
+  auto *info = TypeTable::TryGetTypeInfo(dtype);
+  if (info)
+    return info->name();
+  return "unknown type: " + std::to_string(static_cast<int>(dtype));
+}
+
+inline std::ostream &operator<<(std::ostream &os, DALIDataType dtype) {
+  if (const char *builtin = GetBuiltinTypeName(dtype))
+    return os << builtin;
+  auto *info = TypeTable::TryGetTypeInfo(dtype);
+  if (info)
+    return os << info->name();
+  // Use string concatenation so that the result is the same aas in to_string, unaffected by
```

Comment:
```suggestion
  // Use string concatenation so that the result is the same as in to_string, unaffected by
```

Reply:
Fixed. I'll report anyone who launches CI for that to EPA ;)